src/components/account/details/AccountDetailsPage.js

```diff
@@ -0,0 +1,22 @@
+import React, { Component } from 'react'
+
+import Navigation from '../../dashboard/navigation/Navigation'
+import SideMenu from './SideMenu'
+import DetailsPageMain from './DetailsPageMain'
+
+import '../../../styles/account/details/account_details_page.css'
+
+export default class AccountDetailsPage extends Component {
+  render() {
+    return (
+      <div className="account-details-page">
+        <div className="navigation">
+          <Navigation />
+          <div className="top-border" />
```

Comment:
What's the purpose of an empty div here? Couldn't we apply a style to the container to achieve the same affect (and save some rendering and code)?